voice/asr-outbound-call.py

```diff
@@ -0,0 +1,41 @@
+# Import dependencies
```

Comment:
This script shouldn't be needed (and actually won't even show up on the documentation site), the user should be able to just run the other script and their app can directly access `/webhooks/answer` when a call is forwarded.